pixeltable/functions/huggingface.py

```diff
@@ -85,6 +79,28 @@ def clip_image(image: Batch[PIL.Image.Image], *, model_id: str) -> Batch[np.ndar
     return [embeddings[i] for i in range(embeddings.shape[0])]
 
 
+@pxt.udf(batch_size=32)
+def detr_detections(image: Batch[PIL.Image.Image], *, model_id: str, threshold: float = 0.5) -> Batch[dict]:
```

Comment:
I'm wondering if we should adopt a convention for HF models to use the full model name as the function name, ie, detr_for_object_detection.